src/schemas/server.ts

```diff
@@ -22,11 +22,16 @@
  * SOFTWARE.
  */
 
-import mongoose, { Schema, Document, InferSchemaType } from "mongoose";
+import { Snowflake } from "discord.js";
+import { Schema, HydratedDocument, Model, model } from "mongoose";
 
 const serverSchema: Schema = new Schema({
     _id: String,
-    logs: String,
+    logs: {
+        members: String,
+        messages: String,
+        profiles: String
+    },
```

Comment:
First of all, the server schema has been changed to accommodate moderation logs categories...